scripts/lib/sphinx/generateToc.ts

```diff
@@ -32,7 +32,7 @@ export function generateToc(options: {
     title: string;
     name: string;
     version: string;
-    changelogUrl: string;
+    releaseNotesUrl?: string;
```

Comment:
Why `?`? You always set it in the script.

Reply:
Fixed, thanks!